### ISSUE TITLE
folder created but "Process exited with status 1"

### ISSUE THREAD

**Issue by lasjan** [appleboy/drone-scp#135]
When trying to perform pretty basic upload I'm receiving 
"Process exited with status 1"
However folder c:\Application\CDTEST\APK\artifacts is created.

Config is:
  deploy:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        dotnet-version: ['6.0.x' ]
        project-id: ['CIGitHubTest']
        rid: ['win-x64']
        server-path: ['c:\Application\CDTEST\APK']
    needs: build
    steps:
      - name: echoing
        run: |
            echo "HI"
      - uses: actions/download-artifact@v3
        with:
            name: dotnet-results-${{ matrix.project-id }}-${{ matrix.dotnet-version }}-${{ matrix.rid }}-test
      - name: Display structure of downloaded files
        run: ls -R
      - name: ssh Upload
        uses: appleboy/scp-action@master
        with:
           host: ${{ secrets.CI_SERVER_ADDRES }}
           username: ${{ secrets.CI_GIT_USER }}
           password: ${{ secrets.CI_GIT_PASS }}
           port: ${{ secrets.CI_GIT_PORT }}
           source: release.zip
           target: ${{ matrix.server-path }}\artifacts

Action log is:
/usr/bin/docker run --name f155468f[20](https://github.com/lasjan/ci-test-net/runs/6747224071?check_suite_focus=true#step:6:21)0c7c9eb485a8744f4c1508862a0_e431e5 --label 6f1554 --workdir /github/workspace --rm -e INPUT_HOST -e INPUT_USERNAME -e INPUT_PASSWORD -e INPUT_PORT -e INPUT_SOURCE -e INPUT_TARGET -e INPUT_TIMEOUT -e INPUT_COMMAND_TIMEOUT -e INPUT_KEY -e INPUT_KEY_PATH -e INPUT_PASSPHRASE -e INPUT_FINGERPRINT -e INPUT_USE_INSECURE_CIPHER -e INPUT_RM -e INPUT_DEBUG -e INPUT_STRIP_COMPONENTS -e INPUT_OVERWRITE -e INPUT_TAR_TMP_PATH -e INPUT_PROXY_HOST -e INPUT_PROXY_PORT -e INPUT_PROXY_USERNAME -e INPUT_PROXY_PASSWORD -e INPUT_PROXY_PASSPHRASE -e INPUT_PROXY_TIMEOUT -e INPUT_PROXY_KEY -e INPUT_PROXY_KEY_PATH -e INPUT_PROXY_FINGERPRINT -e INPUT_PROXY_USE_INSECURE_CIPHER -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_REF_NAME -e GITHUB_REF_PROTECTED -e GITHUB_REF_TYPE -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e GITHUB_STEP_SUMMARY -e RUNNER_OS -e RUNNER_ARCH -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/ci-test-net/ci-test-net":"/github/workspace" 6f1554:68f200c7c9eb485a8744f4c1508862a0
tar all files into /tmp/884981472/ICnuQaBwiK.tar
scp file to server.
create folder c:\Application\CDTEST\APK\artifacts
drone-scp error:  Process exited with status 1
drone-scp rollback: remove all target tmp file
remove file ICnuQaBwiK.tar
20***/06/05 20:37:50 Process exited with status 1

**Comment by appleboy** [comment]
@lasjan run scp-action in Windows?

**Comment by Shekhar-Shivam** [comment]
@appleboy  Yes, trying to copy files from Linux Github runner to Windows server. I get this error. Can you explain the error and provide a solution. Thanks

**Comment by lasjan** [comment]
> @lasjan run scp-action in Windows?

Hi, I belive i tried to set this as github action running on github remote server.

**Comment by Elaniobro** [comment]
I too am experiencing the same error/issue as @Shekhar-Shivam:

self hosted github runner on linux and scp files to windows 19 server.

**Comment by appleboy** [comment]
I will take it.

**Comment by appleboy** [comment]
Please try the following version

```diff
  build:
    name: Build
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@master
    - name: copy file via ssh password
-     uses: appleboy/scp-action@master
+     uses: appleboy/scp-action@7af00892de6f8397c5c3393cfb3b32ae7f91b94b
      with:
        host: ${{ secrets.HOST }}
        username: ${{ secrets.USERNAME }}
        password: ${{ secrets.PASSWORD }}
        port: ${{ secrets.PORT }}
        source: "tests/a.txt,tests/b.txt"
        target: "test"
```

**Comment by appleboy** [comment]
fixed in #132
